src/core/plugins/oas3/spec-extensions/wrap-selectors.js

```diff
@@ -32,6 +32,10 @@ const OAS3NullSelector = onlyOAS3(nullSelector)
  * Wrappers
  */
 
+export const findDefinition = onlyOAS3((state, doubledState, schemaName) => (system) => {
```

Comment:
I was getting two states passed to this function despite only passing the schema name, not sure how to change that.

Reply:
Couldn't fix this issue, but perhaps it might have something to do with this https://github.com/swagger-api/swagger-ui/blob/d9baa3dd826f8abbd116ef2e377e5080a03e948f/src/core/system.js#L205